scania-digital-design-system/sdds
# fix(textfield/textarea): changed state prop from string to enum

**Describe pull-request**  
Changed the state prop of the components form string to enum.

**Solving issue**  
Fixes: AB#2746

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Textarea/Textfield
3. Check that the error states works as intended.

**Screenshots**  
-

**Additional context**  
-
